contracts/AirDrop.sol

```diff
@@ -0,0 +1,186 @@
+pragma solidity 0.6.4;
```

Comment:
Because this is totally new contract, we dont use v0.6.4, we can use v0.8.17.

Reply:
fixed